index.js

```diff
@@ -5,6 +5,8 @@ const fileUrl = require('file-url');
 const puppeteer = require('puppeteer');
 const devices = require('puppeteer/DeviceDescriptors');
 const toughCookie = require('tough-cookie');
+const sharp = require('sharp');
```

Comment:
I'm not a big fan of depending on `sharp` as it's a native Node.js addon, which all kinds of problems.

Instead, why can't we just use Canvas in the renderer to combine the images?

Reply:
In node.js, there is no canvas API. Are you suggesting to use this - https://www.npmjs.com/package/canvas
I can use this too. Whatever you suggest !!

Reply:
Are you suggesting the following?
1. Create a local html webpage and open it.
2. The page will load an image(device frame) and the main page in an iframe.
3. Take Screenshot using Canvas API.

Reply:
No, using Puppeteer. You already have a browser at your disposal.

Could maybe use https://developer.mozilla.org/en-US/docs/Web/API/OffscreenCanvas